src/sourcekit-lsp/LanguageClientManager.ts

```diff
@@ -559,6 +560,9 @@ export class LanguageClientManager {
                 })(),
             },
             errorHandler: new SourceKitLSPErrorHandler(5),
+            initializationOptions: {
+                "workspace/peekDocuments": true, // workaround for client capability to handle `PeekDocumentsRequest`
```

Comment:
Is it worthwhile sticking the capability inside another block eg "capabilities", just to separate it from any future initialization options. @ahoppen what do you think? We've discussed in the past using the `initializationOptions` as a way to pass commands for codeActions. 
```suggestion
                "capabilities": {
                    "workspace/peekDocuments": true, // workaround for client capability to handle `PeekDocumentsRequest`
                }
```

Reply:
Because the key of the initialization option is the request method name, there shouldn’t be any conflicts, so I’m fine with the current design.

Reply:
@adam-fowler Yeah, I too believe there won't be any conflicts.